src/parser/jobs/blm/modules/Gauge.tsx

```diff
@@ -575,5 +584,48 @@ export class Gauge extends CoreGauge {
 			}))
 		}
 	}
+
+	private overwriteMessage(timestamp: number) {
+		if (this.overwritePolyglotTimestamps.indexOf(timestamp) > -1) {
+			return <Trans id="blm.gauge.overwrites.polyglot">You overwrote Polyglot</Trans>
+		}
+
+		if (this.overwriteParadoxTimestamps.indexOf(timestamp) > -1) {
+			return <Trans id="blm.gauge.overwrites.paradox">You overwrote <DataLink action="PARADOX"/></Trans>
+		}
+
+		return <Trans id="blm.gauge.overwrites.unknown">Unknown proc overwritten</Trans>
+	}
+
+	override output() {
+		const allOverwrites = this.overwritePolyglotTimestamps.concat(this.overwriteParadoxTimestamps).sort()
+		if (allOverwrites.length === 0) { return false }
+
+		return (
+			<Table collapsing unstackable compact="very">
+				<Table.Header>
+					<Table.Row>
+						<Table.HeaderCell><Trans id="blm.gauge.header.time">Time</Trans></Table.HeaderCell>
+						<Table.HeaderCell><Trans id="blm.gauge.header.event">Event</Trans></Table.HeaderCell>
```

Comment:
I think this column might better be labeled as something like "Overwrote which?", given the other request to leave the title alone. It would also be valid to have the content just be the datalink for the skill in question, either Paradox or Xenoglossy (since we don't really handle AoE as much as single-target).

Reply:
I see a future opportunity to include dropped Enochian timestamps as well - but definitely in a separate changelist. That would mean a more generic header title like "Event" and perhaps more descriptive cell entries.

But for now I'll go with the "Overwrote" header title and keep the notes small.

Reply:
Dropped Enochian can be gotten from the Rotation Outliers table since that's the most serious error the player can make (aside from dying). Adding it elsewhere would be needlessly duplicative, so yeah, let's just keep this table limited to the paradox and Polyglot overwrites.